layouts/country/single.html

```diff
@@ -8,10 +8,14 @@
             {{ partial "sidemenu.html" . }}
         </div>
 
-        <div class="col-12 col-lg-8">
+        <div class="col-12 col-lg-8" data-pagefind-body>
             <div class="o-single__container content_with_heading">
                 <div class="o-single__header">
-                    <h1 data-pagefind-meta="title">{{ .Title }}</h1>
+                    <div class="o-single__header__title">
+                        {{ $logo := .Page.Resources.GetMatch "image.webp" }}
+                        <img data-pagefind-meta="img" src="{{ $logo.RelPermalink }}" alt="Icon" />
```

Comment:
```suggestion
                        <img data-pagefind-meta="img" src="{{ $logo.RelPermalink }}" alt="" />
```
Das Icon bringt hier m. E. keinen Mehrwert, daher kann das alt-Attribut leer bleiben, damit der Screenreader es nicht vorliest :)